lms/djangoapps/course_blocks/api.py

```diff
@@ -23,14 +23,9 @@
 ]
 
 
-_COURSE_BLOCKS_CACHE = None
-
-
 def _get_cache():
-    global _COURSE_BLOCKS_CACHE
-    if not _COURSE_BLOCKS_CACHE:
-        _COURSE_BLOCKS_CACHE = get_cache('lms.course_blocks')
-    return _COURSE_BLOCKS_CACHE
+    """Function exists for mocking/testing, or if we want a custom cache."""
+    return cache
```

Comment:
I wasn't really clear on why the global existed in the first place. Right now, this will just return the default cache.